src/vs/workbench/services/positronConnections/browser/positronConnectionsUtils.ts

```diff
@@ -0,0 +1,104 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2024 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Disposable } from 'vs/base/common/lifecycle';
+import { IPositronConnectionItem } from 'vs/workbench/services/positronConnections/browser/interfaces/positronConnectionsInstance';
+
+export interface IPositronConnectionEntry extends IPositronConnectionItem {
+	/***
+	 * The list of connections entries is flat. Level allows us to find
+	 * how nested an entry is.
+	 */
+	level: number;
+
+	/***
+	 * If the entry is expanded or not. Undefined if the entry is not expandable.
+	 */
+	expanded: boolean | undefined;
```

Comment:
Can be:

```
	/***
	 * If the entry is expanded or not. Undefined if the entry is not expandable.
	 */
	expanded?: boolean;
```

To match `error?: string` below it.

Reply:
Resolved in 6ff77efbc3d0a61d4c9dab516e8fcb3a672a1b42